src/main/java/org/openrewrite/java/testing/assertj/JUnitAssertInstanceOfToAssertThat.java

```diff
@@ -46,25 +46,25 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
         return Preconditions.check(new UsesMethod<>(ASSERT_INSTANCE_OF_MATCHER), new JavaIsoVisitor<ExecutionContext>() {
             @Override
             public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx) {
-                J.MethodInvocation md = super.visitMethodInvocation(method, ctx);
-                if (!ASSERT_INSTANCE_OF_MATCHER.matches(md)) {
-                    return md;
+                J.MethodInvocation mi = super.visitMethodInvocation(method, ctx);
+                if (!ASSERT_INSTANCE_OF_MATCHER.matches(mi)) {
+                    return mi;
                 }
 
-                maybeAddImport("org.assertj.core.api.Assertions", "assertThat", false);
+                maybeAddImport("org.assertj.core.api.Assertions", "assertThat");
```

Comment:
Appreciate this fix and the rename of the variable here to meet the conventions, thanks!

Reply:
Well to be honest, I forgot to apply this everywhere. Once I did, there was one test that failed: `importAddedForCustomArguments` (JUnitAssertNotNullToAssertThatTest). As that test also has an issue number assigned to it (#419), it's probably important... 

So, do we want the `onlyIfReferenced` flag on or not?

Reply:
Hmm yes let's keep that; we need it because our templates here are intentionally not context sensitive (would take a lot longer if we can not cache the templates for each time we make a change), and it's fine here to force add the import as we are guaranteed to use `assertThat` after the template returns.

Reply:
👍🏻, I will put it back in for all other recipes (the other recipe's do simply not have such a test, but I guess the same rule applies to all).